packages/config/src/validate/validations.js

```diff
@@ -91,6 +91,14 @@ const POST_NORMALIZE_VALIDATIONS = [
     message: 'must be a string.',
     example: () => ({ plugins: [{ package: 'netlify-plugin-one' }] }),
   },
+  // We don't allow `package@tag|version` nor `git:...`, `github:...`,
+  // `https://...`, etc.
+  {
+    property: 'plugins.*.package',
+    check: package => (package.startsWith('@') || !package.includes('@')) && !package.includes(':'),
```

Comment:
I think `@sentry/netlify-build-plugin@1.0.0` will pass this validation

Reply:
Right! Fixed.